### PR TITLE
Fixed absTime and element inspector bugs

### LINKED ISSUE #537: chartVal undefined for monotonic positive and satisfied
And for monotonic negative and denied. Should not be able to select this option

Screen Shot 2021-08-10 at 3 19 44 PM

### PR DESCRIPTION
**Pull request by YesugenB**
Closes #537 and #535

### CODE REVIEW

**Review by Domily11 — DISMISSED** [review]
Once you fix the bugs I mentioned you are good to go!!

**Review by i-m-foster — DISMISSED** [review]

File: leaf-ui/rappid-extensions/BackboneModelsIntention.js [hunk 1]
```diff
@@ -229,15 +229,16 @@ var IntentionBBM = Backbone.Model.extend({
         
         if (this.get('evolvingFunction') != null) {
             var funcSegList = this.getFuncSegments();
-            // If the function is C or UD & C set refEvidencePair to initValue
-            if (this.get('evolvingFunction').get('type') == 'C') { 
-                    funcSegList[0].set('refEvidencePair', initValue); // Set first index of funcSegList to given initValue 
-            }
-            else {
+            // // If the function is C or UD & C set refEvidencePair to initValue
```
Comment (amgrubb) [inline leaf-ui/rappid-extensions/BackboneModelsIntention.js]: Can this commented out code be removed?

File: leaf-ui/rappid-extensions/BackboneModelsIntention.js [hunk 2]
```diff
@@ -229,15 +229,16 @@ var IntentionBBM = Backbone.Model.extend({
...
+            // if (this.get('evolvingFunction').get('type') == 'C') { 
+            //         funcSegList[0].set('refEvidencePair', initValue); // Set first index of funcSegList to given initValue 
+            // }
+            // else {
             // If the function is not C or UD & C, reset the function type and clear the functionSegList
             this.get('evolvingFunction').set('type', 'NT');
             this.get('evolvingFunction').set('functionSegList', []);
-            }
+            console.log(this)
```
Comment (amgrubb) [inline leaf-ui/rappid-extensions/BackboneModelsIntention.js]: Remove console log?

File: leaf-ui/rappid-extensions/ElementInspector.js [hunk 3]
```diff
@@ -148,6 +148,7 @@ var ElementInspector = Backbone.View.extend({
      * Initializes the element inspector using previously defined templates
      */
     render: function () {
+       // console.log(this.intention)
```
Comment (amgrubb) [inline leaf-ui/rappid-extensions/ElementInspector.js]: remove this line

**Review by amgrubb — CHANGES_REQUESTED** [review]
A couple of small changes to clean up the code. Otherwise looks good.

**Review by i-m-foster — APPROVED** [review]

**Review by amgrubb — APPROVED** [review]

**Pull request merged**